hash/apple2_flop_clcracked.xml

```diff
@@ -50384,7 +50384,7 @@ license:CC0-1.0
 		</part>
 	</software>
 
-	<software name="pfsfa00">
+	<software name="pfsfa00" cloneof="pfsfile">
```

Comment:
For cases like this, can you please just make the `name` of the clone the name of the parent with a suffix appended?  It makes it easier to see the groups of clones in a sorted listing.  In this case, you can make the clone’s `name` `pfsfilea00`.  There’s no need to force the `name` for clones into eight characters (up to sixteen characters are allowed by validation).